src/containers/SearchPage/components/form/SearchContentForm.jsx

```diff
@@ -93,6 +95,8 @@ class SearchContentForm extends Component {
       subjects,
       query,
       users,
+      language: lang,
```

Comment:
Bruker lang som navn i search-object for at den ikkje skal dukke opp i statuslista. Kun aktuellt for innholdsøk.